### PR TITLE
Remove backslash from multi line comments

### PR DESCRIPTION
**Pull request by baojun-nervana**
paddle has issues with multi line comments with backslash

paddle/build/third_party/install/ngraph/include/ngraph/check.hpp:133:1: error: multi-line comment [-Werror=comment]
// The "..." may be filled with expressions of any type that has an "operator<<" overload for\

### CODE REVIEW

File: src/ngraph/check.hpp [hunk 1]
```diff
@@ -130,8 +130,8 @@ namespace ngraph
 //
 // Take care to fully qualify the exception class name in the macro body.
 //
-// The "..." may be filled with expressions of any type that has an "operator<<" overload for\
-// insertion into std::ostream.
+// The "..." may be filled with expressions of any type that has an "operator<<" overload<backslash>
```
Comment (diyessi) [inline src/ngraph/check.hpp]: Can you instead break the line so `overload/insertion` is in the next line?

File: src/ngraph/check.hpp [hunk 2]
```diff
@@ -130,8 +130,8 @@ namespace ngraph
 //
 // Take care to fully qualify the exception class name in the macro body.
 //
-// The "..." may be filled with expressions of any type that has an "operator<<" overload for\
-// insertion into std::ostream.
+// The "..." may be filled with expressions of any type that has an "operator<<"        <backslash>
+// overload for insertion into std::ostream.
 //
```
Comment (diyessi) [inline src/ngraph/check.hpp]: Now I see what this is trying to say. I suggest:
`// The "..." may be filled with expressions of any type that has an "operator<<" `
`// overload that inserts backslashes into std::ostream`

Comment (aprocter) [reply src/ngraph/check.hpp]: That’s not what it was trying to say. The backslash didn’t belong there in the first place, not sure how it happened. It should just be deleted.

**Comment by silee2** [comment]
Duplicate of #2780: Closing

**Pull request closed without merging**